### PR TITLE
code add category hotels and create template add category hotels

### PR DESCRIPTION
**Pull request by at-tuannd**

### CODE REVIEW

File: app/Http/Controllers/Admin/CategoryHotelController.php [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        if (Auth::user()->level==1) {
```
Comment (at-vietpb) [inline app/Http/Controllers/Admin/CategoryHotelController.php]: What is mean value `1` please define this value in config const.

Comment (at-tuannd) [inline app/Http/Controllers/Admin/CategoryHotelController.php]: yes. thank you

File: app/Http/Controllers/Admin/CategoryHotelController.php [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+use Request;
+
+class CategoryHotelController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return void
+     */
+    protected $cHotelRepositories;
```
Comment (at-vietpb) [inline app/Http/Controllers/Admin/CategoryHotelController.php]: This variable for CategoryHotel or Hotel? Please make clear variable name

File: app/Http/Controllers/Admin/CategoryHotelController.php [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param int $id [descriptions]
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $edit=$this->cHotelRepositories->find($id);
```
Comment (at-vietpb) [inline app/Http/Controllers/Admin/CategoryHotelController.php]: You should make name variable simple and clear. Don't keep `$edit`

File: app/Http/Controllers/Admin/CategoryHotelController.php (hunk 2 above)
Comment (at-tuannd) [inline app/Http/Controllers/Admin/CategoryHotelController.php]: yes. thank you

File: app/Http/Controllers/Admin/CategoryHotelController.php (hunk 3 above)
Comment (at-tuannd) [inline app/Http/Controllers/Admin/CategoryHotelController.php]: yes. thank you

File: app/Http/Controllers/Admin/CategoryHotelController.php [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    /**
+     * Remove the specified resource from storage.
+     *
+     * @param int $id [descriptions]
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
+    {
+        $this->cHotelRepositories->deleteCategoryHotel($id);
```
Comment (at-vietpb) [inline app/Http/Controllers/Admin/CategoryHotelController.php]: What happened with hotel after delete category?

File: app/Repositories/Admin/CategoryHotelRepository.php [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+     *
+     * @param Id $id [description]
+     *
+     * @return void
+     */
+    public function deleteCategoryHotel($id)
+    {
+        try {
+            DB::beginTransaction();
+            $delete=$this->model->find($id);
```
Comment (at-vietpb) [inline app/Repositories/Admin/CategoryHotelRepository.php]: Please make clear variable name

File: app/Http/Controllers/Admin/CategoryHotelController.php [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+    /**
+     * Remove the specified resource from storage.
+     *
+     * @param int $id [descriptions]
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
+    {
+        $this->categoryHotel->deleteCategoryHotel($id);
```
Comment (at-vietpb) [inline app/Http/Controllers/Admin/CategoryHotelController.php]: What happened with hotels after delete category?

File: app/Http/Controllers/Admin/CategoryHotelController.php (hunk 4 above)
Comment (at-tuannd) [inline app/Http/Controllers/Admin/CategoryHotelController.php]: delete category hotel and remove hotel. config in Models/Hotel

File: app/Repositories/Admin/CategoryHotelRepository.php (hunk 5 above)
Comment (at-tuannd) [inline app/Repositories/Admin/CategoryHotelRepository.php]: yes. Thank you

File: app/Http/Controllers/Admin/CategoryHotelController.php (hunk 6 above)
Comment (at-tuannd) [inline app/Http/Controllers/Admin/CategoryHotelController.php]: config file Models/Hotel.php

**Pull request merged**